core/src/test/java/org/wildfly/extras/sunstone/api/impl/docker/NodeLeakOnBadPortsTest.java

```diff
@@ -0,0 +1,42 @@
+package org.wildfly.extras.sunstone.api.impl.docker;
+
+import org.jclouds.compute.ComputeService;
+import org.jclouds.compute.domain.ComputeMetadata;
+import org.junit.Assert;
+import org.junit.Test;
+import org.wildfly.extras.sunstone.api.CloudProperties;
+import org.wildfly.extras.sunstone.api.CloudProvider;
+import org.wildfly.extras.sunstone.api.impl.AbstractJCloudsCloudProvider;
+
+import java.util.Optional;
+
+public class NodeLeakOnBadPortsTest {
+
+    @Test
+    public void test() {
+
+        CloudProperties.getInstance().reset().load(this.getClass());
+        CloudProvider cloudProvider = CloudProvider.create("provider0");
+
+        try {
+            cloudProvider.createNode("node0");
+            Assert.fail("Port 2468 was unexpectedly opened");
+        } catch (Exception e) {
+            if (cloudProvider instanceof AbstractJCloudsCloudProvider) {
+                ComputeService computeService = ((AbstractJCloudsCloudProvider) cloudProvider).getComputeServiceContext().getComputeService();
+                Optional<? extends ComputeMetadata> latestNode = computeService.listNodes().stream().findFirst();
```

Comment:
Suggested change is in LittleJohnII/sunstone#1 

We could move it into the `AbstractCloudProviderTest` in the future.